.github/workflows/odh-kubeflow-release-tag.yaml

```
@@ -0,0 +1,74 @@
+env:
+  REPO_ORG: opendatahub-io
+  REPO_NAME: kubeflow
+
+permissions:
+  contents: write
+
+jobs:
+  create-release:
+    name: Create GitHub Release
+    runs-on: ubuntu-latest
+    if: startsWith(github.event.head_commit.message, 'chore: release ')
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)